js/main.js

```diff
@@ -7,50 +7,13 @@ $(document).ready(() => {
     $("#copyBtn").on("click", copyInstallCmd);
     $("#copyCmdBtn").on("click", copyGenerateTextCmd);
 
-    let titleList = ["ElasticSearch", "Logging"];
-    let data = [
-      {
-        title: "ElasticSearch",
-        // basicIntro:"Basic intro",
-        detailedList: [
-          {
-            title: "",
-            linkList: [
-              {
-                itemHead: "Elasticsearch Index and Replica Management",
-                itemTime: "6 min",
-                redirectPage: "./guides/elasticSearch.html",
-              }
-            ],
-          }
-        ],
-      },
-      {
-        title: "Logging",
-        // basicIntro: "Basic intro",
-        detailedList: [
-          {
-            title: "",
-            linkList: [
-              {
-                itemHead: "Logging Frameworks in Python",
-                itemTime: "6 min",
-                redirectPage: "./guides/Logging-frameworks-python.html",
-              },
-              {
-                itemHead: "Logging Frameworks in Go",
-                itemTime: "4 min",
-              },
-            ],
-          }
-        ],
-      },
-    ];
-    $("#guide-link-list").guide({
-      titleList: titleList,
-      data: data,
-    });
-
+    const currentUrl = window.location.href;
+    if (currentUrl.includes("guides.html")) {
+      renderGuidesList();
+    }
+    // else if (currentUrl.includes("questions.html")) {
+    //   renderQuestionsList();
+    // }
```

Comment:
Remove the commented code

Reply:
This part removes the fake data for questions. I'll leave it as is since it helps to understand the format of the questions data.